src/site/labs/iterators/iterators.rst

```diff
@@ -49,6 +49,11 @@ Create an Iterable Stack
     * Refer to the ``toString`` from the :download:`ArraySortedBag <../../../main/java/ArraySortedBag.java>` for help
 
 
+.. note::
+
+    Be mindful of the order that the elements in the ``ArrayStack`` are returned by the iterator. 
```

Comment:
This reads really hard.

The order of the elements in the stack are returned by the iterator -- I have to do mental gymnastics to infer its say "yo, order brother."

Reply:
Rephrasing it as a question might be more understandable. Asking, _in what order should the iterator produce elements_?